migrations/20231009074209_cw20_activity_add_tx_hash.ts

```diff
@@ -0,0 +1,43 @@
+import { Knex } from 'knex';
+import { Cw20Event } from '../src/models';
+import _ from 'lodash';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('cw20_activity', (table) => {
+    table.string('tx_hash').defaultTo('');
+  });
+  await knex.transaction(async (trx) => {
+    let currentId = 0;
+    while (true) {
+      const cw20Activities = await Cw20Event.query()
+        .transacting(trx)
+        .joinRelated('event.message.transaction')
+        .where('event.id', '>', currentId)
+        .orderBy('event.id', 'asc')
+        .limit(1000)
+        .select(
+          'cw20_activity.*',
+          'event:message:transaction.hash as tx_hash',
```

Comment:
chắc lấy tx_id thôi, chứ cái bảng này có khi cũng to đấy

Reply:
Em xử lý 1000 records mỗi lần ạ